atomkraft/test/__init__.py

```diff
@@ -37,15 +48,18 @@ def RequiredFileOption(help, default):
 def trace(
     # currently, require the trace to be present.
     # later, there will be an option to pick up the last one from the model
-    trace: Optional[Path] = FileOption("trace to execute", "model"),
+    trace: Optional[Path] = PathOption(
```

Comment:
I believe we agreed to name this option `--path`... `--trace` was fine for a single file; for a directory this name is confusing

Reply:
Renamed `--trace` to `--path` :slightly_smiling_face: